input/fsh/VS_Other.fsh

```diff
@@ -107,173 +107,10 @@ Description: "Includes surgical procedure codes from SNOMED CT, ICD-10-PCS and C
 
 ValueSet:       TumorMarkerTestVS
 Id: mcode-tumor-marker-test-vs
-Description:    "Codes representing tests for tumor markers. This value set of LOINC codes is not comprehensive and can be extended. LOINC codes are preferred. Other vocabularies can be used only if the test of interest is not covered by LOINC.
+Description:    "Codes representing tests for tumor markers. This value set of LOINC codes is not comprehensive and can be extended. Other vocabularies can be used only if the test of interest is not covered by LOINC. Tumor marker tests differ from genetic tests in that they measure at the levels of the protein and substance post-RNA protein synthesis."
```

Comment:
I think the end of this sentence is confusing. Here's a possible alternative, but I'm not sure I hit the mark.

```suggestion
Description:    "Codes representing tests for tumor markers. This value set of LOINC codes is not comprehensive and can be extended. Other vocabularies can be used only if the test of interest is not covered by LOINC. Tumor marker tests differ from genetic tests in that they measure levels of protein or other substances produced downstream from RNA protein synthesis."
```